models/slashes.go

```diff
@@ -0,0 +1,10 @@
+package models
```

Comment:
Please move this to `providers/transip`.  While this function seems re-usable by other providers, I'd rather have each provider be independent as much as possible.